class/defaults.yml

```diff
@@ -1,4 +1,42 @@
 parameters:
   openshift4_s3_forwarder:
     =_metadata: {}
-    namespace: syn-openshift4-s3-forwarder
+    multi_instance: true
+    namespace: openshift-logging
+
+    images:
+      fluentd:
+        registry: docker.io
```

Comment:
Is there a way to use quay.io? The rate-limit might cause harm.

Reply:
Yes, that should be possible. Currently i deployed to docker.io bc all the credentials where already present in the project/group.
This can easaly changed later.